modules/optimization/doc/content/modules/optimization/examples/material_transient.md

```diff
@@ -0,0 +1,89 @@
+
+- [Theory](theory/InvOptTheory.md)
+- [Examples overview](optimization/examples/index.md)
+- [Example 1: Convective Boundary Conditions](materialInv_ConvectiveBC.md)
+- [Example 2: Constant Thermal Conductivity](materialInv_ConstK.md)
+- [Example 3: Transient Solve with Automatic Adjoint](material_transient.md)
+- [debuggingHelp.md]
+
+# Example: Transient Solve with automatic adjoing problem id=sec:TransientSolveAutomaticAdjoint
+
+In this example, material parameters are optimized within a transient solve setting. This is accomplished
+by leveraging inverse optimization theory, see [section](theory/InvOptTheory.md#sec:material_inversion). As
+with [material inversion with constant conductivity](materialInv_ConstK.md), this is a nonlinear optimization
+problem that requires an appropriate initial guess and parameter bounds to obtain convergence.
+
+## Transient Problem Definition
+
+The problem analyzed here is a two-dimensional diffusion example with square geometry. The right
+and top boundaries are subjected to zero Dirichlet boundary conditions. The spatial distribution of
+the material parameter $D$ is given in [diff_parameter]. While this material parameter is known in the forward
+problem, its distribution to minimize the solution misfit is optimized in the adjoint pass.
+
+
+!media large_media/optimization/material_transient_diff.png
+       style=width:40%;margin:auto;padding-top:2.5%;background-color: white;color: black;
+       id=diff_parameter
+       caption=Spatial parameterization of the diffusivity parameter for the transient problem.
+
+As this is a transient or non-steady problem, the solution at each time step depends on the solution
+at previous steps. The forward problem is solved as usual in MOOSE, using implicit Euler time stepping.
+The solution to the diffusion problem is obtained for ten steps with a time step of 0.1 seconds. The
+solution to the diffused variable at time $t=0.6$ s, together with the measurement points provided for
+material inversion optimization, is shown in [solution].
+
+!media large_media/optimization/material_transient_solution.png
+       style=width:40%;margin:auto;padding-top:2.5%;background-color: white;color: black;
+       id=solution
+       caption=Snaphot of the forward simulation at the end of the sixth time step (0.6 s). The yellow
+       marking points to the measurement points provided in the procedure for obtaining
+       the optimal diffusivity parameters of the material at the end of the adjoint problem.
+
+The solution to the adjoint problem requires an initial solution vector with estimates of the material
+property values. In addition to that, bounds for those parameter need to be provided. In [sensitivity],
+we plot the objective function as we vary the initial estimate for the diffusion parameter at the location
```

Comment:
```suggestion
we plot the objective function as we vary the thermal conductivity at the location
```

Reply:
Are we varying the thermal conductivity or just the estimate? The actual thermal conductivity would be given by the material property used to solve the forward problem?

Reply:
We are varying the estimate for the thermal conductivity but that estimate is what is being used in the forward simulation.  Does this make sense?

Reply:
Yeah, it does. But I think that the input to the algorithm is the estimate, so that may be more relevant to the user and his/her interpretation of the plot.